src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/IndexStateManagementRestTestCase.kt

```diff
@@ -516,6 +517,40 @@ abstract class IndexStateManagementRestTestCase : ESRestTestCase() {
         return true
     }
 
+    protected fun assertIndexExists(index: String) {
+        val response = client().makeRequest("HEAD", index)
+        assertEquals("Index $index does not exist.", RestStatus.OK, response.restStatus())
+    }
+
+    protected fun assertIndexDoesNotExist(index: String) {
+        val response = client().makeRequest("HEAD", index)
+        assertEquals("Index $index does not exist.", RestStatus.NOT_FOUND, response.restStatus())
```

Comment:
NP: Assertion message here should be printing out "Index ```does``` exist" if the response status is not ```RestStatus.NOT_FOUND``` I believe.

Reply:
fixed.